ui/src/notebooks/pipes/Data/AggregateWindowSelector.tsx

```diff
@@ -10,12 +10,18 @@ import {PipeContext} from 'src/notebooks/context/pipe'
 // Constants
 import {FUNCTIONS, QueryFn} from 'src/timeMachine/constants/queryBuilder'
 
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+
 const AggregateFunctionSelector: FC = () => {
   const {data, update} = useContext(PipeContext)
   const selectedFunction = data?.aggregateFunction || FUNCTIONS[0]
 
   const updateSelectedFunction = useCallback(
     (aggregateFunction: QueryFn): void => {
+      event(
+        `Updating the Aggregate function in the Flow Query Builder to ${aggregateFunction.name}`
```

Comment:
we should put the function name as a tag on the event so that we can compare across event instances. iirc:
```
event(`Updating the Aggregate function in the Flow Query Builder`, { function: aggregateFunction.name })
```

Reply:
Sounds good